README.md

```diff
@@ -117,7 +117,7 @@ We will use the [VAE outlier detector](https://docs.seldon.io/projects/alibi-det
 
 ```python
 from alibi_detect.od import OutlierVAE
-from alibi_detect.utils import save_detector, load_detector
+from alibi_detect.utils.saving import save_detector, load_detector
```

Comment:
Can be changed to use the public api `alibi_detect.saving`, although not strictly part of this PR (can be a different one).